core/pkg/runtime/from_config.go

```diff
@@ -82,6 +82,11 @@ func FromConfig(logger *logger.Logger, version string, config Config) (*Runtime,
 		return nil, err
 	}
 
+	options, err := telemetry.BuildConnectOptions(telCfg)
```

Comment:
IMHO we should swallow the error, warn the user, and proceed anyhow. Telemetry is nice to have but not a blocker to operate flagd

Reply:
Yeag this is a fair point 🤔  I will choose to log and continue then

Reply:
hmm I checked other places where we have telemetry errors (specially on set-up phase), and strategy is to fail when there're errors (see - https://github.com/open-feature/flagd/blob/main/core/pkg/runtime/from_config.go#L47-L57).

If I change this now, we will be breaking this pattern. So I think this is something to discuss and change in a future iteration 🤔 

cc - @beeme1mr & @toddbaert

Reply:
Agreed. If @thisthat is right we should make the change in a different PR and be consistent. You can create an investigation issue if you want.

Reply:
I created a follow-up tasl - https://github.com/open-feature/flagd/issues/1194

@thisthat If you are satisfied, please resolve this conversation and approve the PR :)